docs/enterprise-guide/saml-auth0.md

```diff
@@ -0,0 +1,69 @@
+# Setting up SAML with Auth0
+
+Setting up Metabase with Auth0 is extremely easy. Just follow this configuration guide and you'll be ready to go in a few couple of minutes.
```

Comment:
```suggestion
```

Avoid saying anything is easy in docs. Because people might run into situations that make it more complicated, or they have trouble following the guide, or whatever, and it's not easy for them. They get frustrated and we lose trust.

Reply:
got it!